README.md

```diff
@@ -17,6 +17,22 @@ In order to build and serve the web site locally, run :
     bundle install
     bundle exec jekyll serve
 
+Once running the website is accessibly at `localhost:4000`.
+
+## Build using a container
+
+To avoid needing to install Ruby you can build and run the web site locally inside a container, run:
+
+    docker run --rm -v ${PWD}:/srv/jekyll -p 4000:4000 -it jekyll/jekyll jekyll serve
+
+It can take several minutes to complete, look for the message "Server running...".
+
+**Note:** If you see the error `require': cannot load such file -- webrick (LoadError)` add the following to the `Gemfile`:
```

Comment:
```suggestion
**Note:** If you see the error `require': cannot load such file -- webrick (LoadError)`. add the following to the `Gemfile`:
```

Reply:
I assumed this suggestion should be a comma not a period so have added that